spec/core/player_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rspec'
+require 'UI/UI'
+require 'core/player'
+require 'core/board'
+
+describe Player do
+
+  before(:each) do
+    @board = Board.new(3)
+    @output = StringIO.new
+  end
+
+  it 'plays her/his turn' do
```

Comment:
The only thing I can think with this test is that it has a lot of other class dependencies and it's all testing the output of the UI as well. Is there a way to more directly test whatever player.move is doing?